ruma-events/src/relation.rs

```diff
@@ -0,0 +1,43 @@
+//! Types describing event relations after MSC 2674, 2675, 2676, 2677.
+
+use std::fmt::Debug;
+
+use js_int::{Int, UInt};
+use serde::{Deserialize, Serialize};
+
+/// Summary of all reactions with the given key to an event.
+#[derive(Clone, Debug, Default, Deserialize, PartialEq, Eq, Serialize)]
+pub struct BundledReaction {
+    /// The key (emoji) used for reaction.
+    pub key: String,
+    /// Time of the bundled reaction being compiled on the server.
+    pub origin_server_ts: Option<Int>,
+    /// Number of times it was reacted with the key to the event.
```

Comment:
I don't understand what this is trying to say. Is this supposed to be the number of reactions?